src/main/resources/api.yaml

```diff
@@ -264,6 +264,45 @@ paths:
                 $ref: '#/components/schemas/SprintDto'
 
   /projects/{projectId}/sprints/{sprintId}:
+    get:
+      tags:
+        - projects
+        - sprints
+      summary: Obtains a sprint from a project by its id and state
+      operationId: getSprint
+      parameters:
+        - in: path
+          name: projectId
+          schema:
+            type: integer
+            format: int64
+          required: true
+          description: Numeric ID of the project
+        - in: path
+          name: sprintId
+          schema:
+            type: integer
+            format: int64
+          required: true
+          description: Numeric ID of the sprint
+        - in: query
+          name: sprintStates
```

Comment:
niezbyt rozumiem sens podawania id i stanu, mogłabyś wyjaśnić?

Reply:
Nie wiem, to był eksperyment, niekoniecznie ma sens. W sensie jak ta stara metoda przestaje zwracać listę tylko zaczyna zwracać pierwszy sprint w jednym z tych stanów z ustawionym nextem i prevem, to żeby sobie przeiterować po wszytstkich sprintach w danym stanie, musisz ifować za każdym razem, czy nie wyjechałeś poza ten stan. Trochę mi się to nie podobało, ale pewnie jest okej i po prostu mi żal mojego kodu z backendu :stuck_out_tongue: 

Można faktycznie zrobić `getFollowing` i `getPreceding` i w nich zachować te stany? "Daj mi kolejny w stanie x"?